Only in dwmblocks-async/: build
diff -up dwmblocks-async/config.h dwmblocks-origin/config.h
--- dwmblocks-async/config.h	2025-09-03 18:12:22.668828227 -0400
+++ dwmblocks-origin/config.h	2025-09-07 18:21:05.854409199 -0400
@@ -2,30 +2,31 @@
 #define CONFIG_H
 
 // String used to delimit block outputs in the status.
-#define DELIMITER " | "
+#define DELIMITER "  "
 
 // Maximum number of Unicode characters that a block can output.
 #define MAX_BLOCK_OUTPUT_LENGTH 45
 
 // Control whether blocks are clickable.
-#define CLICKABLE_BLOCKS 0
+#define CLICKABLE_BLOCKS 1
 
 // Control whether a leading delimiter should be prepended to the status.
-#define LEADING_DELIMITER 1
+#define LEADING_DELIMITER 0
 
 // Control whether a trailing delimiter should be appended to the status.
-#define TRAILING_DELIMITER 1
+#define TRAILING_DELIMITER 0
 
 // Define blocks for the status feed as X(icon, cmd, interval, signal).
 #define BLOCKS(X)             \
-    X("", "sh_do-not-disturb", 0, 1) \
-    X("", "sh_vpn",            0, 2) \
-    X("", "sh_timedate",       1, 0) \
-    X("", "sh_brightness",     0, 3) \
-    X("", "sh_volume",         0, 4) \
-    X("", "sh_network",       10, 0) \
-    X("", "sh_battery",        5, 0) \
-    X("", "sh_memory",        10, 0) \
-    X("", "sh_loadavg",        5, 0)
+    X("", "sb-mail", 600, 1)  \
+    X("", "sb-music", 0, 2)   \
+    X("", "sb-disk", 1800, 3) \
+    X("", "sb-memory", 10, 4) \
+    X("", "sb-loadavg", 5, 5) \
+    X("", "sb-mic", 0, 6)     \
+    X("", "sb-record", 0, 7)  \
+    X("", "sb-volume", 0, 8)  \
+    X("", "sb-battery", 5, 9) \
+    X("", "sb-date", 1, 10)
 
 #endif  // CONFIG_H
Common subdirectories: dwmblocks-async/.git and dwmblocks-origin/.git
Common subdirectories: dwmblocks-async/.github and dwmblocks-origin/.github
Common subdirectories: dwmblocks-async/include and dwmblocks-origin/include
Common subdirectories: dwmblocks-async/src and dwmblocks-origin/src
